src/code/utilities/lang/en-us.js

```diff
@@ -1,5 +1,8 @@
 export default {
   // Challenge instructions
+  "~FV_EGG_GAME.BREED_BUTTON": "Breed",
```

Comment:
Already mentioned above, but since the design has the label capitalized, I think it should be capitalized here rather than in code.